src/components/KillTeam2021/data/index.tsx

```diff
@@ -0,0 +1,12 @@
+import veteranGuardsmen from "./veteranGuardsmen"
+
+const getData = (factionName: string) => {
+  switch (factionName) {
+    case "Veteran Guardsmen":
```

Comment:
Ah, there's a `camelCase` method in `lodash`, which we're already using, so no need for a new dependency: https://lodash.com/docs/4.17.15#camelCase

Reply:
@Floppy Can you share more about how the solution with `camelCase` would work? I think I'm not seeing what you are seeing, because the only thing coming to my mind is doing dynamic import, but that would make the function async, and that would be tricky to deal with, I think?